hooks/types.ts

```diff
@@ -0,0 +1,14 @@
+type CMSItem = {
+  id: number;
+  user_created: string;
```

Comment:
I would argue that we should hide the item's metadata just in case. At least for the user IDs `user_created` and `user_updated`